kaizen/cli/cli.py

```
@@ -374,5 +378,93 @@ def sync_phoenix(
+        try:
+            with zipfile.ZipFile(output_file, "w", zipfile.ZIP_DEFLATED) as zf:
+                for file_path in skill_path.rglob("*"):
+                    if file_path.is_file():
+                        # Archive path includes skill name as top-level directory
+                        arcname = f"{skill_name}/{file_path.relative_to(skill_path)}"
+                        zf.write(file_path, arcname)
+
+            console.print(f"[green]Packaged:[/green] {skill_name} -> {output_file}")
+            packaged += 1
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)